Bencodex.Tests/Types/DictionaryTest.cs

```diff
@@ -74,5 +75,89 @@ public void Equality()
             });
             Assert.NotEqual(a, c);
         }
+
+        [Fact]
+        public void SetItem()
+        {
+            var dictionary = Dictionary.Empty
+                .SetItem("text", "foo")
+                .SetItem("integer", 1337)
+                .SetItem("binary", new byte[] { 0x01, 0x02, 0x03, 0x04 })
+                .SetItem("boolean", true)
+                .SetItem("list", new IValue[] { (Text) "bar", (Integer) 1337 });
+
+            Assert.Equal("foo", (Text) dictionary["text"]);
+            Assert.Equal("foo", dictionary.GetValue<Text>("text"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Integer>("text"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Binary>("text"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Boolean>("text"));
+
+            Assert.Equal((Integer) 1337, (Integer) dictionary["integer"]);
+            Assert.Equal((Integer) 1337, dictionary.GetValue<Integer>("integer"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Binary>("integer"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Text>("integer"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Boolean>("integer"));
+
+            Assert.Equal(new byte[] { 0x01, 0x02, 0x03, 0x04 }, (Binary) dictionary["binary"]);
+            Assert.Equal(new byte[] { 0x01, 0x02, 0x03, 0x04 }, dictionary.GetValue<Binary>("binary"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Integer>("binary"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Text>("binary"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Boolean>("binary"));
+
+            Assert.Equal((Bencodex.Types.Boolean) true, (Bencodex.Types.Boolean) dictionary["boolean"]);
+            Assert.Equal((Bencodex.Types.Boolean) true, dictionary.GetValue<Boolean>("boolean"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Integer>("boolean"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Binary>("boolean"));
+            Assert.Throws<InvalidCastException>(() => dictionary.GetValue<Text>("boolean"));
+
+            Assert.Equal(new IValue[] { (Text)"bar", (Integer)1337 }, (Bencodex.Types.List) dictionary["list"]);
+            Assert.Equal(new IValue[] { (Text)"bar", (Integer)1337 }, dictionary.GetValue<Bencodex.Types.List>("list"));
```

Comment:
Some lines like this seem too long.  FYI at the time I started this project, I maintained every line is not over 80 characters.  I agree that it is not a good standard for C# projects, but anyway could you make the code style consistent to what it had been?

Reply:
I agree about your opinion. Also it seems to need to set up editorcofnig.